### PR TITLE
Changed SDF constructor to accept more convenient data types.

### PR DESCRIPTION
**Pull request by luisenp**

### CODE REVIEW

File: theseus/embodied/collision/signed_distance_field.py [hunk 1]
```diff
@@ -71,37 +82,75 @@ def _compute_sdf_data_from_map(
...
         return sdf_data_var
 
-    def update_data(self, origin: Variable, sdf_data: Variable, cell_size: Variable):
-        if sdf_data.ndim != 3:
-            raise ValueError(
-                "Argument sdf_data to SignedDistanceField2D must be a batch of matrices."
-            )
-        if not (origin.ndim == 2 or (origin.ndim == 3 and origin.shape[2] == 1)):
+    @staticmethod
+    def convert_origin(origin: Union[torch.Tensor, Point2]) -> Point2:
```
Comment (mhmukadam) [inline theseus/embodied/collision/signed_distance_field.py:96]: Are these convert functions meant to be private and start with a trailing underscore?

File: theseus/embodied/collision/tests/utils.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import torch
```
Comment (mhmukadam) [inline theseus/embodied/collision/tests/utils.py:1]: Missing license header. Also `theseus/embodied/collision/tests/__init__.py` is missing license hearder.

**Review by mhmukadam — APPROVED** [review]
LGTM!

File: theseus/embodied/collision/signed_distance_field.py (hunk 1 above)
Comment (luisenp) [reply theseus/embodied/collision/signed_distance_field.py:96]: I'm also using these in the two collision cost functions.

File: theseus/embodied/collision/tests/utils.py (hunk 2 above)
Comment (luisenp) [reply theseus/embodied/collision/tests/utils.py:6]: Good catch! Fixed.

**Review by mhmukadam — APPROVED** [review]

**Pull request merged**